### ISSUE TITLE
Look Into Porting BOINC to KaiOS

### ISSUE THREAD

**Issue by RoboticMind** [BOINC/boinc#2622]
KaiOS is looking to be a possible third contender in the mobile market. Google just invested 22 Million USD in them, see the article from [TechCrunch.](https://techcrunch.com/2018/06/27/google-kaios/) As stated in that article there are more than 40 million phones running KaiOS at the moment
>KaiOS tells us that there have been more than 40 million KaiOS phones shipped to-date

An important thing to know about KaiOS is that it is designed for low-end feature phones that might not make this worth any effort until we see large adoption. This is probably something to look into in the future and maybe keep in mind.

Regardless, I think it could be worth reaching out to them about BOINC now so that we good relations with them and hear their thoughts. 

Here is their official [website](https://www.kaiostech.com)

**Comment by RoboticMind** [comment]
I think adding support here might be possible through #3086 since it looks like it runs apps only through the browser

**Comment by AenBleidd** [comment]
I took a look at this OS. Yes, it runs of Linux, but it uses Gecko (basically web-browser) engine to run applications.
BOINC currently doesn't run on any browser, and taking into account the very tiny market share of KaiOS (0.13% as of June 2024(, it doesn't make sense to put any efforts to support this OS.
![image](https://github.com/user-attachments/assets/2cb9d05c-5b94-4cad-821c-74276a29d229)
Closing this for now without any further actions.
